### PR TITLE
5580 vardump output

### PR DESCRIPTION
**Pull request by EdwardAngert**
**Note:** Please fill out the PR Template to ensure proper processing.

Closes: #5580 

## Summary

**[Reading Pantheon Environment Configuration](https://pantheon.io/docs/read-environment-config)** - Truncates the example output of the var_dump so as not to seem exhaustive. [PR 5590]

## Effect

The following changes are already committed:

- md fixes
- Truncates the example output of the var_dump so as not to seem exhaustive.

## Remaining Work

The following changes still need to be completed:

- [x] +1 from @ari-gold 
- [x] Checkover Review from @pantheon-systems/docs-admins 

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5605 | Difference |
| --- | --- | --- | --- |
| Performance Score | `51` | `48` | `-3` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5605` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19375/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19375/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/read-environment-config.md [hunk 1]
```diff
@@ -110,57 +100,59 @@ The `$_SERVER` variable contains sensitive data about a site and should not be p
...
+    string(62) "/srv/bindings/non-static-binding-string-inserted-here/code//index.php"
+    ["HTTP_HOST"]=>
+    string(25) "example-wp-site.pantheonsite.io"
+    ["HTTP_USER_AGENT"]=>
+    string(120) "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko)         Chrome/46.0.2490.80 Safari/537.36"
+    ["HTTP_ACCEPT"]=>
+    string(74) "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8"
+    ["HTTP_ACCEPT_LANGUAGE"]=>
+    string(14) "en-US,en;q=0.8"
+    etc...
```
Comment (ari-gold) [inline source/content/read-environment-config.md]: this one has "etc" and the first doesn't? Do we want  them  to match for consistency?

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5605 | Difference |
| --- | --- | --- | --- |
| Performance Score | `64` | `49` | `-15` |
| Best Practice Score | `93` | `79` | `-14` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `96` | `-2` |

Lighthouse Audit Reports:

- [`pr-5605` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19385/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19385/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/read-environment-config.md (hunk 1 above)
Comment (EdwardAngert) [reply source/content/read-environment-config.md]: agreed. thank you!
fixed and marking as resolved

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5605 | Difference |
| --- | --- | --- | --- |
| Performance Score | `54` | `56.99999999999999` | `3` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5605` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19389/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19389/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5605 | Difference |
| --- | --- | --- | --- |
| Performance Score | `54` | `46` | `-8` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5605` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19423/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/19423/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**